src/morphology_workflows/annotation.py

```diff
@@ -263,21 +263,21 @@ def plot_cut_leaves(row, data_dir, with_plotly=True):
             plot_path = (data_dir / row.name).with_suffix(".html")
             markers.plot(filename=plot_path)
     else:  # pragma: no cover
-        plt.figure()
-        plot_path = (data_dir / row.name).with_suffix(".pdf")
-        neuron = load_morphology(row.morph_path)
-        view.plot_morph(
-            neuron, plt.gca(), realistic_diameters=False, plane="xz", soma_outline=False
-        )
-
         if markers:
+            plt.figure()
```

Comment:
So in the end you could rework the `if` order like this:
```python
if markers:
    if with_plotly:
        ...
    else:
        ...
```

Reply:
ah yes indeed